lib/go-atscfg/chkconfig.go

```diff
@@ -26,17 +26,35 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+// ChkconfigFileName is the name of the ckconfig configuration file.
 const ChkconfigFileName = `chkconfig`
+
+// ChkconfigParamConfigFile is the ConfigFile value of Parameters that affect
+// the generation of the chkconfig configuration file.
 const ChkconfigParamConfigFile = `chkconfig`
+
+// ContentTypeChkconfig is the MIME content type of the contents of the
+// chkconfig configuration file.
+//
+// TODO: The GoDoc for MakeChkconfig says "This is a JSON object, and should be
+// served with an 'application/json' Content-Type." - is that right? Or is this
+// right? Or are they both right, somehow?
 const ContentTypeChkconfig = ContentTypeTextASCII
+
+// LineCommentChkconfig is the string that signifies the start of a line comment
+// in the grammar of a chkconfig configuration file.
 const LineCommentChkconfig = LineCommentHash
 
 // ChkconfigOpts contains settings to configure generation options.
 type ChkconfigOpts struct {
 }
 
 // MakeChkconfig returns the 'chkconfig' ATS config file endpoint.
-// This is a JSON object, and should be served with an 'application/json' Content-Type.
+//
+// This is a JSON object, and should be served with an 'application/json'
+// Content-Type.
+//
+// TODO: rename/rework? We systemd now, after all.
```

Comment:
We plan to. I think we're leaning toward removing it altogether, and recommending people manage that with a system management tool (like Ansible or Puppet) rather than ATC.